Fix tag access

christophberger opened the pull request:
Hi,
Thank you for this super-useful tool. I only found it did not fetch any tags (they all were `null`). Seems the takeout format has changed. I had to add a second `children[0]` to the tag access code to be able to fetch the tags. 

Please review and merge if you agree to the changes.

vHanda commented:
Hey. Thanks for the pull request.

This seems to be breaking the existing test, though. Do you think you could support both the old and the new format?

christophberger commented:
Hey, sorry for the late update, this got totally out of my focus.
I changed the code to be able to deal with both the "classic" tags and the ones I encountered, and I also added a test for the new tags structure.

I see the CI tests are passing now.

Please review and let me know if there is anything else left to do, thanks

vHanda commented:
Thanks a lot. This is awesome.
